Jbarget/oxymore
# erotic stories minor adjustments to layout and spacing

### What changes have you made?
- cleaned up the spacing a bit on the scrollable text to make more use of the space in the grid cells, it's a subtle change but visually it should look easier on the eye

### Which issue(s) does this PR fix?

Fixes #13 

### Screenshots (if there are design changes)


### How to test
- Set the timer back to a date in the past and the content should display 
- Make sure there are no oversights with spacing on mobile or desktop view 
- The two stories should still scroll smoothly when you reach the end of the first, the page should scroll to the next
